app/services/mailer/transactional_email.rb

```diff
@@ -0,0 +1,82 @@
+require 'mandrill'
+
+class TransactionalEmail
+  attr_reader :mandrill, :template_name, :brand, :merge_params, :message
+
+  module Types
+    WELCOME     = 'welcome'
+    PLAN_CHANGE = 'plan-change'
+    CANCEL_PLAN = 'cancel-plan'
+  end
+
+  def initialize(template_name, brand)
```

Comment:
I think we should be passing in a third argument, something like `merge_params`. That will make this email service more extensible as we will be able to pass in as many merge vars as we like.